panther_analysis_tool/main.py

```diff
@@ -1805,15 +1806,25 @@ def parse_filter(filters: List[str]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
 
 
 def run() -> None:
+    # setup logger and print version info as necessary
+    logging.basicConfig(
+        format="[%(levelname)s]: %(message)s",
+        level=logging.INFO,
+    )
+    if not pat_utils.is_latest():
+        logging.warning(
+            "A new version of %s is available. To upgrade, run:\n\tpip3 install %s --upgrade\n",
```

Comment:
Do we want to include the current and new version numbers, like the pulumi message does?

Reply:
updated, lmk what you think